app.R

```diff
@@ -1,49 +1,104 @@
-#
-# This is a Shiny web application. You can run the application by clicking
-# the 'Run App' button above.
-#
-# Find out more about building applications with Shiny here:
-#
-#    http://shiny.rstudio.com/
-#
 
 library(shiny)
+library(tidyverse)
+library(readr)
+
+setwd("~/Desktop/390/SCBI-ForestGEO-Data/tree_main_census/data/census-csv-files/")
```

Comment:
The shiny app code is not replicable b/c of the filepath. Take a look at the [Lec07](https://rudeboybert.github.io/SDS390/index.html#Lec_07:_Tue_922) video on filepaths, use the `data/` folder, and commit these changes and then I'll be able to help with your code issue

Reply:
Also, your GitHub repo should be self-contained and portable so that anyone can recreate your work. So any necessary .csv files should be included.